client/components/accordion/status.jsx

```diff
@@ -0,0 +1,87 @@
+/**
+ * External dependencies
+ */
+import React, { PureComponent, PropTypes } from 'react';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'components/gridicon';
+import Tooltip from 'components/tooltip';
+
+/**
+ * Constants
+ */
+const STATUS_GRIDICON = {
+	info: 'info',
+	warning: 'notice',
+	error: 'notice'
+};
+
+export default class AccordionStatus extends PureComponent {
+	static propTypes = {
+		type: PropTypes.oneOf( [
+			'info',
+			'warning',
+			'error'
+		] ),
+		text: PropTypes.node,
+		url: PropTypes.string,
+		position: PropTypes.string,
+		onClick: PropTypes.func
+	};
+
+	static defaultProps = {
+		type: 'info',
+		onClick: () => {}
+	};
+
+	state = {};
```

Comment:
>Is this needed?

Missed this comment earlier. We use state taking advantage that it will trigger a re-render and attach the `Tooltip` to the anchor.